### ISSUE TITLE
preload-lcp-image audit is not consistent

### ISSUE THREAD

**Issue by kevinfarrugia** [GoogleChrome/lighthouse#12580]
#### Provide the steps to reproduce
1. Run LH on https://kevinfarrugia.github.io/resource-hints/preload-lcp-image-script

#### What is the current behavior?
According to the definition in https://github.com/GoogleChrome/lighthouse/blob/master/lighthouse-core/audits/preload-lcp-image.js, Lighthouse audits should show a warning for "Preload Largest Contentful Paint image" when the LCP image is not preloaded, using the following conditions:
- If it's already preloaded, no need to recommend it.    
- It's not a request loaded over the network, don't recommend it.    
- It's already discoverable from the main document, don't recommend it.
- Finally, return whether or not it belongs to the main frame

In the demo URL provided above, the image `./img/banner.jpg` is fetched by the deferred script `./preload-lcp-image-script.js` and therefore should fail the audit. However, while it occasionally fails the audit, this behaviour is not consistent and more often than not, it is treated as a passed audit.

#### What is the expected behavior?
If the image is not preloaded (and meets the above conditions), then the audit `preload-lcp-image` should fail in 100% of cases.

#### Environment Information
* Affected Channels: DevTools, PageSpeed Insights
* Lighthouse version: 7.3.0
* Chrome version: 90.0.4430.97
* Operating System: Ubuntu 20.04/OSX Big Sur

**Comment by patrickhulce** [comment]
Thanks for filing with such research @kevinfarrugia!

Those are necessary conditions, but they aren't sufficient for the suggestion to be there. A simulation of the page load with the preload in place *also* needs to find substantive savings. This simulation can be affected by small differences (see [variability docs](https://github.com/GoogleChrome/lighthouse/blob/master/docs/variability.md)) and it's very possible to sometimes recommend the action and sometimes not. In those cases, it means the amount of savings possible by preloading this image is very close to the threshold for ignoring it. The correct interpretation for this Lighthouse behavior is that this recommendation will improve things for some of your page loads, and for others it might not.

In other words...

> If the image is not preloaded (and meets the above conditions), then the audit preload-lcp-image should fail in 100% of cases.

is not the intended expectation.

**Comment by kevinfarrugia** [comment]
Hi @patrickhulce , thank you for your quick and detailed reply. Are the thresholds in terms of `wastedMs`? If so, are these documented please?

Thanks again.

**Comment by patrickhulce** [comment]
> Are the thresholds in terms of wastedMs

Yep!
https://github.com/GoogleChrome/lighthouse/blob/9f58f30393d960917d2c04d4df354d8207d84b3b/lighthouse-core/audits/byte-efficiency/byte-efficiency-audit.js#L46-L59

> If so, are these documented please?

Not publicly. We try not to publicize internal implementation details that shouldn't affect user action. (i.e. the threshold of 300ms being orange for this audit is not a magic immovable number that's always right or significant, subject to change at any time).

**Comment by kevinfarrugia** [comment]
Perfect, thank you for your feedback. 🙏
